src/client/src/index.tsx

```diff
@@ -11,9 +11,28 @@ const SimpleLauncher = lazy(() => import('./apps/SimpleLauncher'))
 
 const urlParams = new URLSearchParams(window.location.search)
 
+function getEnv() {
+  const serviceUrl = process.env.REACT_APP_BROKER_HOST || window.location.host;
+  if (serviceUrl.includes('localhost')) return 'localhost';
+  const envMatch = serviceUrl.match(/web-([a-zA-Z]+)\.adaptivecluster\.com/);
```

Comment:
@mattydono with regex's I tend to prefer a named capture group as opposed to using array index matching.

Reply:
this has now been updated in latest commit and is significantly easier to read as opposed to the index matching!